### DISCUSSION TITLE
Mailchimp Action

### DISCUSSION THREAD

**Discussion by tobimori** [tobimori/kirby-dreamform#3] [Feature Requests]
- Subscribe users to a specific list on a form submission, and map form fields to Mailchimp fields
https://mailchimp.com/developer/

**Comment by tobimori** [comment]
Will be included in 1.2.0

**Comment by silllli** [comment]
Maybe a stupid question, but was this action removed in a later release?

**Reply by tobimori** [reply]
No, but you're right they're no docs for it. Sorry for the inconvenience. If you set the API key in the config the action will show:
https://github.com/tobimori/kirby-dreamform/blob/develop/config/options.php#L106

**Reply by silllli** [reply]
Ah, thanks, I will have a look!
